packages/hagrid/hagrid/cli.py

```diff
@@ -2621,6 +2621,10 @@ def run_quickstart(
         venv_dir = directory + ".venv"
         environ["PATH"] = venv_dir + os.sep + os_bin_path + os.pathsep + environ["PATH"]
         jupyter_binary = "jupyter.exe" if is_windows() else "jupyter"
+
+        env_activate_cmd = "source " + venv_dir + os.sep + os_bin_path + "/activate"
```

Comment:
This is also different for windows cmd.exe or powershell.

Reply:
Fixed and tested.